### ISSUE TITLE
React Native support

### ISSUE THREAD

**Issue by vitalyrotari** [solkimicreb/react-easy-state#17]
Hi, I just test this library in React Native 0.53.0, and it's work like a charm. If anyone can test on lower version and give some feedback this will be very nice.

**Comment by solkimicreb** [comment]
Hi, I guess you are testing it on iOS. This is the current status - as for as I know. 

React Native uses the JavaScriptCore engine, which is built into iOS. iOS 10 and above does support Proxies so Easy State works there.

Android however doesn't have JavaScriptCore and RN devs decided to ship a version of it instead of using different engines on different platforms. The sad news is that the JavaScriptCore engine for Android was not updated for more than 3 years now. This means that any non transpilable feature form the last 3 years will not work on Android phones.

So in a nutshell Easy State should work on iOS and in Chrome debugger, but NOT in Android. I am not 100% sure as there are surprisingly scarce news about it. I would be happy to be proven wrong 🙂

**Comment by vitalyrotari** [comment]
@solkimicreb Thanks for alternative JavaScriptCore for Android, need to try it.

**Comment by solkimicreb** [comment]
Let me know if it was simple to set up or a nightmare 😄

**Comment by vitalyrotari** [comment]
@solkimicreb Just test it on Android with custom JavaScriptCore for Android, it's work like a charm ;) Implementation of JavaScriptCore is easy, just follow the instructions.

**Comment by solkimicreb** [comment]
Thanks a lot, this is awesome news! 🙂 

I will modify the readme a bit to reflect this. Closing the issue now, feel free to reopen it at any time.

Thanks for the help again!
